lib/livebook/application.ex

```diff
@@ -260,8 +260,14 @@ defmodule Livebook.Application do
     cond do
       teams_key && auth ->
         case String.split(auth, ":") do
-          ["offline", name, public_key] -> create_offline_hub(teams_key, name, public_key)
-          _ -> Livebook.Config.abort!("Invalid LIVEBOOK_TEAMS_AUTH configuration.")
+          ["offline", name, public_key] ->
+            create_offline_hub(teams_key, name, public_key)
+
+          ["online", name, org_id, org_key_id, agent_key] ->
```

Comment:
Is there a reason we pass agent name in a separate env var, given we pass the agent key here?

Reply:
I'm following the same structure for `LIVEBOOK_TEAMS_AUTH`, but yeah we could pass the agent name here.

Reply:
In this one, the hub name and the agent name can come from the server, no? So here we need `org_id:org_key_id:agent_key` only?

Reply:
I think for this one the hub name can come from the server, right? As well as the org public key?

Reply:
It should be a separate variable because we want to allow the user to change `LIVEBOOK_AGENT_NAME` to whatever they want but they should not mess with `LIVEBOOK_TEAMS_AUTH`.